resharper/resharper-unity/src/ProjectModel/MetaFileTracker.cs

```diff
@@ -21,19 +21,19 @@ public class MetaFileTracker : IChangeProvider
         private readonly ILogger myLogger;
         private IProjectItem myLastAddedItem;
 
-        public MetaFileTracker(Lifetime lifetime, ChangeManager changeManager, ISolution solution, ILogger logger, 
-            ISolutionLoadTasksScheduler solutionLoadTasksScheduler,
-            UnitySolutionTracker unitySolutionTracker)
+        public MetaFileTracker(Lifetime lifetime, ChangeManager changeManager, ISolution solution, ILogger logger,
+                               ISolutionLoadTasksScheduler solutionLoadTasksScheduler,
+                               UnitySolutionTracker unitySolutionTracker)
         {
             mySolution = solution;
             myLogger = logger;
 
             solutionLoadTasksScheduler.EnqueueTask(new SolutionLoadTask("AdviseForChanges", SolutionLoadTaskKinds.AfterDone,
                 () =>
                 {
-                    if (!unitySolutionTracker.IsUnityGeneratedProject.Value)
+                    if (!unitySolutionTracker.IsUnityProjectFolder.Value)
```

Comment:
Is this intentional? You have already changed `IsUnityGeneratedProject` to accept wrong casing.
I am affraid of custom valid solution in UnityProject folder, which may include some non-generated projects.

Reply:
Yeah, intentional. The casing issue is still a problem we should fix, even if we don't use it for meta files. And I think if we're changing files in the `Assets` or `Packages` folders with a custom solution, then we should handle meta files too. We know it's a Unity solution, so we should do our best not to break things.

Reply:
Ok, I see your point.